src/System.Globalization/tests/RegionInfo/RegionInfoTests.Properties.cs

```diff
@@ -12,7 +12,12 @@ public class RegionInfoPropertyTests
         [Fact]
         public void CurrentRegion()
         {
+#if WindowsDesktop
```

Comment:
All these #ifdefs tell me that we actually have a lot of inconsistencies between the target frameworks we should actually take a close look at these to be sure we are OK with these differences.